core/domain/question_fetchers_test.py

```diff
@@ -26,85 +26,98 @@
 
-        self.question_id = question_services.get_new_question_id()
-        self.question = self.save_new_question(
+        self.question_id = question_services.get_new_question_id() # type: ignore[no-untyped-call]
+        self.question = self.save_new_question( # type: ignore[no-untyped-call]
             self.question_id, self.editor_id,
-            self._create_valid_question_data('ABC'), ['skill_1'])
+            self._create_valid_question_data('ABC'), ['skill_1']) # type: ignore[no-untyped-call]
 
-    def test_get_questions_and_skill_descriptions_by_skill_ids(self):
-        question_services.create_new_question_skill_link(
+    def test_get_questions_and_skill_descriptions_by_skill_ids(self) -> None:
+        question_services.create_new_question_skill_link( # type: ignore[no-untyped-call]
             self.editor_id, self.question_id, 'skill_1', 0.3)
 
         questions, _ = (
             question_fetchers.get_questions_and_skill_descriptions_by_skill_ids(
                 2, ['skill_1'], 0))
 
         self.assertEqual(len(questions), 1)
+        assert isinstance(questions[0], question_domain.Question)
         self.assertEqual(
-            questions[0].to_dict(), self.question.to_dict())
+            questions[0].to_dict(), self.question.to_dict()) # type: ignore[no-untyped-call]
 
-    def test_get_no_questions_with_no_skill_ids(self):
+    def test_get_no_questions_with_no_skill_ids(self) -> None:
         questions, _ = (
             question_fetchers.get_questions_and_skill_descriptions_by_skill_ids(
                 1, [], 0))
         self.assertEqual(len(questions), 0)
 
-    def test_get_questions_with_multi_skill_ids(self):
-        question_id_1 = question_services.get_new_question_id()
-        question_1 = self.save_new_question(
+    def test_get_questions_with_multi_skill_ids(self) -> None:
+        question_id_1 = question_services.get_new_question_id() # type: ignore[no-untyped-call]
+        question_1 = self.save_new_question( # type: ignore[no-untyped-call]
             question_id_1, self.editor_id,
-            self._create_valid_question_data('ABC'), ['skill_1', 'skill_2'])
-        question_services.create_new_question_skill_link(
+            self._create_valid_question_data('ABC'), # type: ignore[no-untyped-call]
+                ['skill_1', 'skill_2'])
+        question_services.create_new_question_skill_link( # type: ignore[no-untyped-call]
             self.editor_id, question_id_1, 'skill_1', 0.3)
-        question_services.create_new_question_skill_link(
+        question_services.create_new_question_skill_link( # type: ignore[no-untyped-call]
             self.editor_id, question_id_1, 'skill_2', 0.5)
 
         questions, _ = (
             question_fetchers.get_questions_and_skill_descriptions_by_skill_ids(
                 2, ['skill_1', 'skill_2'], 0))
 
         self.assertEqual(len(questions), 1)
+        assert isinstance(questions[0], question_domain.Question)
```

Comment:
Can we add a comment here on why this assertion statement is here. More of like `# Ruling out the possibility of None for mypy type checking.`

Ditto elsewhere assertion has been used of ruling out mypy errors

Reply:
Done